pkg/transfers/inbound/cleanup_test.go

```diff
@@ -10,12 +10,41 @@ import (
 	"os"
 	"path/filepath"
 	"testing"
-
-	"github.com/moov-io/paygate/pkg/upload"
+	"time"
 
 	"github.com/moov-io/base/log"
+	"github.com/moov-io/paygate/pkg/upload"
 )
 
+type mockedFS struct {
+	fileSize int64
+	modTime  time.Time
+}
+
+func (m mockedFS) Name() string {
+	panic("implement me")
```

Comment:
Should these just return "dummy" values, like `return "name"` instead of panicing?

Reply:
They totally should even though they aren't being used. I'll update these.